modules/collect/src/test/java/com/opengamma/collect/OptionalDoubleAssert.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (C) 2014 - present by OpenGamma Inc. and the OpenGamma group of companies
```

Comment:
The year is wrong. Copy-paste or do you need to update your IntelliJ file template?